.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,307 @@
+name: Build Windows
+
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        type: string
+        default: windows-latest
+      tag:
+        type: string
+        required: true
+      arch:
+        type: string
+        required: true
+      cpu:
+        type: string
+        required: true
+      assertions:
+        type: boolean
+        default: false
+      deoptimize:
+        type: boolean
```

Comment:
is there a reason this cannot be named `optimize` and be a string? with the default being ReleaseFast?